### DISCUSSION TITLE
Is there a way to have winget take over updating apps installed outside of winget?

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [microsoft/winget-cli#2596] [Q&A]
Is there a way to have winget take over updating apps installed outside of winget?

**Comment by Masamune3210** [comment]
If you mean automatically disabling auto updates in specific apps, no. That is handled in the apps themselves as its impossible to know what myriad of apps you might have installed and how to safely modify all their individual settings to disable auto update.
